### ISSUE TITLE
Arithmetic operations

### ISSUE THREAD

**Issue by Prathik4** [Forty-Tw0/RaspberryPi-FreeRTOS#2]
First of all thanks for providing port for raspberry pi 2 model.

I'm facing problems while doing floating point arithmetic operations.
I can't add or subtract or multiply the float or double primitive type variables.
Can you please help me out and tell me what must be causing this?

**Comment by Forty-Tw0** [comment]
I recall having some trouble getting floating point operations to work on the hardware, so I just set a compiler flag to use software floating point libraries to 'fix' a compiler error.
-mfloat-abi=
https://github.com/Forty-Tw0/RaspberryPi-FreeRTOS/blob/f484e68ec9deaa0afaec96ba5779b368793f0c4d/dbuild.config.mk#L10

And I just realised both hard and softfp are defined, that may or may not be a problem. It probably comes down to what the toolchain knows how to do.

http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0773b/chr1383143713787.html
